app/models/shelter.rb

```diff
@@ -1,5 +1,6 @@
 class Shelter < ApplicationRecord
   has_many :pets
+  has_many :shelter_reviews
```

Comment:
Added this as part of creating the Shelter Reviews class and defining relationship between shelter and reviews